src/mad_spark_alt/cli.py

```
@@ -596,6 +601,13 @@ async def _run_evolution_pipeline(
                 console.print(
                     f"• Best from generation: {metrics.get('best_fitness_generation', 0)}"
                 )
+                
+                # Show cache performance if available
+                cache_stats = metrics.get('cache_stats')
+                if cache_stats and cache_stats.get('hits', 0) > 0:
+                    console.print(f"\n[cyan]💾 Cache Performance:[/cyan]")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)